backend/dev-env/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1
-FROM python:3.9-slim-bullseye
+FROM python:3.9-slim-bookworm
```

Comment:
I would prefer to avoid that change, to stay consistent with the version used on production. There must be something else going on to explain why it works in the CI on `main` and not on your system :thinking: 

Is it possible that your system use an older version of the image `python:3.9-slim-bullseye` for some reason? Have you tried `docker pull python:3.9-slim-bullseye` ?

Reply:
> I would prefer to avoid that change, to stay consistent with the version used on production.

That's a good reason indeed.

> Have you tried `docker pull python:3.9-slim-bullseye` ?

It indeed fixed the problem.

Reply:
Ok nice. It's not clear why this is needed though. On my system it seems that `docker compose up --build` will try to update the base image. I suppose it may depend on whether the specific tag was explicitly pulled on the system before, or on the `docker compose` version itself.